ffmpeg/ffmpeg_test.go

```diff
@@ -28,12 +28,14 @@ func setupTest(t *testing.T) (func(cmd string), string) {
 	// Executes the given bash script and checks the results.
 	// The script is passed two arguments:
 	// a tempdir and the current working directory.
-	cmdFunc := func(cmd string) {
+	cmdFunc := func(cmd string) bool {
```

Comment:
Do we need to return a bool here if the cmd string executed via `exec.Command()` triggers an error if it ends with an exit code of 1?

See [this test example](https://github.com/livepeer/lpms/blob/5c05cbbeb53bf7cfbc413c5b33ddadf3d3699719/ffmpeg/ffmpeg_test.go#L107) that I believes will cause the test to fail if the bash command using ffprobe has an exit code of 1.

Reply:
Ah right so `t.Error()` will log the error, but if the caller of the command wants to know if the command failed it currently doesn't have a way to know that and returning a bool solves that. Got it.